### ISSUE TITLE
DateEntry doesn't work on Mac OSX default theme

### ISSUE THREAD

**Issue by 18melissa40** [j4321/tkcalendar#16]
Clicking on the DateEntry widget's drop-down menu will not produce the drop-down calendar on Mac OSX using the default aqua theme. There is no error message, but nothing happens. Reading the changelog, it looks like this was previously an issue on Windows but has been fixed. However, it is still not working on Mac. Using the "clam" theme does work, however.

**Comment by j4321** [comment]
I don't have a Mac, so I won't be able to test my code. But I might have an idea of what the problem is but I'll need your help to check it. Can you run the following code, click on the DateEntry button and tell me what is printed in the console?

```
from tkcalendar import DateEntry
from tkinter import Tk

root = Tk()

d = DateEntry(root)
d.bind('<1>', lambda e: print(d.identify(e.x, e.y)))
d.pack()
root.mainloop()

```

**Comment by j4321** [comment]
I think the code from branch [fix-#16](https://github.com/j4321/tkcalendar/commits/fix-%2316) fixes your issue. Let me know if it does work.
